src/krait/utils/update.py

```diff
@@ -69,14 +70,12 @@ def should_check_update(ctx: click.Context) -> bool:
 
 def run_python_command(cmd: str) -> str:
     try:
-        o = subprocess.check_output([
+        return subprocess.check_output([
             sys.executable,
             *cmd.split()
         ], stderr=subprocess.DEVNULL).decode('utf-8')
-    except Exception:
-        o = ''
-
-    return o
+    except subprocess.CalledProcessError:  # Non-zero exit code
+        return ''
```

Comment:
The more general exception was purposeful here; want to ensure that any other issue that occurs gets caught (like decoding issues etc) and don't get relayed back to user